src/main/java/someboty/Managers/dateManager.java

```diff
@@ -0,0 +1,77 @@
+package someboty.Managers;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import someboty.Exceptions.InputException;
+
+/**
+ * dateManager handles processing and formatting of date and time.
+ * This class does not need to be initialized to use its methods and so
+ * does not have a constructor.
+ */
+public class DateManager {
+
+    private static final DateTimeFormatter OUT_FORMATT = DateTimeFormatter.ofPattern("dd MMM yyyy HH:mm");
+
+    // accepted date formats
+    private static final String[] DATE_FORMATS = {
+        "yyyy-MM-dd HHmm",
+        "dd-MM-yyyy HHmm",
+        "MM-dd-yyyy HHmm",
+        "dd/MM/yyyy HHmm",
+        "MM/dd/yyyy HHmm",
+        "yyyy/MM/dd HHmm",
+        "dd MMM yyyy HH:mm"
+    };
```

Comment:
Your method of making date formatting more robust is pretty neat. It allows code to be more resilient to various date formats so nice job